packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -55,6 +55,12 @@ export class Combobox extends FormAssociatedCombobox {
     @attr({ attribute: "autocomplete", mode: "fromView" })
     autocomplete: ComboboxAutocomplete | undefined;
 
+    /**
+     * @internal
+     */
+    @observable
+    public slottedLabel: Node[];
```

Comment:
A note here - we likely won't be able to create 100% consistency across all form elements w/ labels in their implementation because in certain scenarios like checkbox and radio, the label *is* the default slotted content. In a scenario like this where _options_ make more sense, we will need a different pattern. This Pr proposes naming the slot "label" when a name is required and following this pattern for the slotted directive.